base/bin/migration.mustache

```diff
@@ -7,16 +7,17 @@ const migrationName = `${now}_${process.argv[2]}`;
 const opts = { migrationName };
 
 const migrationTemplate = fs.readFileSync('./bin/migration_template.mustache').toString();
-const migrationTestTemplate = fs.readFileSync('./bin/migration_test_template.mustache').toString();
+const migrationTestTemplate = fs.readFileSync('./bin/migration_template_test.mustache').toString();
 
 const migrationFile = mustache.render(migrationTemplate, opts);
 const migrationTest = mustache.render(migrationTestTemplate, opts);
 
 try {
-  fs.mkdirSync('./tests/migrations/');
+  fs.mkdirSync('./migrations/');
+  fs.mkdirSync('./test/migrations/');
```

Comment:
This will fail unexpectedly if `migrations` already exists but `test/migrations` doesn't.